covalent_dispatcher/_cli/service.py

```diff
@@ -410,12 +447,11 @@ def logs() -> None:
     Show Covalent server logs.
     """
     if os.path.exists(UI_LOGFILE):
-        f = open(UI_LOGFILE, "r")
-        line = f.readline()
-        while line:
-            click.echo(line.rstrip("\n"))
+        with open(UI_LOGFILE, "r") as f:
             line = f.readline()
-        f.close()
+            while line:
+                click.echo(line.rstrip("\n"))
+                line = f.readline()
     else:
         click.echo(f"{UI_LOGFILE} not found!. Server possibly purged!")
```

Comment:
Edit message to say "{UI_LOGFILE} not found. Restart the server to create a new log file."